apps/synapse-portal-framework/src/components/DetailsPage/DetailsPage.test.tsx

```diff
@@ -1,310 +1,9 @@
-import React from 'react'
-import DetailsPage, { SplitStringToComponent } from './DetailsPage'
-import { render, screen, waitFor } from '@testing-library/react'
-import userEvent from '@testing-library/user-event'
-import {
-  DetailsPageTabProps,
-  RowSynapseConfig,
-} from '../../types/portal-util-types'
-import syn16787123Json from '../../mocks/syn16787123.json'
-import {
-  AsynchronousJobStatus,
-  ColumnMultiValueFunction,
-  EntityHeader,
-  PaginatedResults,
-  QueryBundleRequest,
-  QueryResultBundle,
-} from '@sage-bionetworks/synapse-types'
-import * as SynapseComponentModule from '../../components/SynapseComponent'
-import { MemoryRouter } from 'react-router-dom'
-import {
-  FullContextProvider,
-  SynapseClientError,
-  SynapseQueries,
-} from 'synapse-react-client'
-import { describe, expect, it, vi } from 'vitest'
-import { UseQueryResult } from '@tanstack/react-query'
-
-vi.mock('synapse-react-client', async importActual => {
-  const actual = await importActual<typeof import('synapse-react-client')>()
-  return {
-    ...actual,
-    SynapseQueries: {
-      ...actual.SynapseQueries,
-      useGetEntityHeaders: vi.fn(),
-      useGetQueryResultBundleWithAsyncStatus: vi.fn(),
-    },
-  }
-})
-
-function renderWithContext(component) {
-  return render(
-    <MemoryRouter>
-      <FullContextProvider
-        synapseContext={{
-          accessToken: 'abcd',
-          utcTime: false,
-          isInExperimentalMode: false,
-          downloadCartPageUrl: '/DownloadCart',
-        }}
-      >
-        {component}
-      </FullContextProvider>
-    </MemoryRouter>,
-  )
-}
-
-const mockGetQueryTableResults = vi.mocked(
-  SynapseQueries.useGetQueryResultBundleWithAsyncStatus,
-)
-const mockGetEntityHeaders = vi.mocked(SynapseQueries.useGetEntityHeaders)
-
-mockGetQueryTableResults.mockResolvedValue({
-  isLoading: false,
-  error: null,
-  data: {
-    jobState: 'COMPLETE',
-    responseBody: syn16787123Json as QueryResultBundle,
-  } as AsynchronousJobStatus<QueryBundleRequest, QueryResultBundle>,
-} as UseQueryResult<AsynchronousJobStatus<QueryBundleRequest, QueryResultBundle>, SynapseClientError>)
-
-const expected: PaginatedResults<EntityHeader> = {
-  results: [
-    {
-      id: 'syn26843747',
-      name: 'My Entity',
-      type: 'org.sagebionetworks.repo.model.FileEntity',
-      versionNumber: 1,
-      versionLabel: '1',
-      benefactorId: 122,
-      createdOn: 'today',
-      modifiedOn: 'earlier',
-      createdBy: 'me',
-      modifiedBy: 'you',
-      isLatestVersion: true,
-    },
-  ],
-}
-mockGetEntityHeaders.mockResolvedValue({ data: expected } as UseQueryResult<
-  PaginatedResults<EntityHeader>,
-  SynapseClientError
->)
+import { describe, it } from 'vitest'
 
 describe('DetailsPage tests', () => {
```

Comment:
The previous set of DetailsPage tests basically covered routing logic that we pulled out of DetailsPage and offloaded to React Router, so there's no point in trying to preserve them.

The DetailsPage still queries a table and passes the data to its descendants, which should be captured in tests (eventually...)